pkg/cmd/roachprod/main.go

```diff
@@ -1159,6 +1159,54 @@ var grafanaURLCmd = &cobra.Command{
 	}),
 }
 
+var grafanaAnnotationCmd = &cobra.Command{
+	Use:   `grafana-annotation <host> <text> --secure --tags [<tag1>, ...] --dashboard-uid <dashboard-uid> --time-range [<start-time> <end-time>]`,
```

Comment:
Nit: might be good to include the comma here in the `time-rage` flag for clarity.